.github/CONTRIBUTING.md

```diff
@@ -1,20 +1,49 @@
-# Documentation
+# Contributing
 
-We use a fork and pull model, you can read about
-[forking repos at github help](https://help.github.com/articles/fork-a-repo/).
+We use [GitHub](https://github.com/ElektraInitiative/libelektra/) to maintain this project. First of all you will need a GitHub account and [Git](https://www.git-scm.com/).
+If you are looking for a IDE you can use [CLion](https://www.jetbrains.com/clion/) ([setup tutorial](/doc/tutorials/contributing-clion.md)) or [Visual Studio Code](https://code.visualstudio.com/).
```

Comment:
I would rephrase "you can use" as anybody is free to use anything. But instead something like "it is known to work" or "current Elektra developers use".